app/controllers/file_controller.rb

```diff
@@ -17,18 +17,19 @@ def show
 
   private
 
+  def allowed_params
+    params.permit(:id, :file_name)
+  end
+  alias stacks_file_params allowed_params
+
```

Comment:
Perhaps we can add a comment here, just so a future developer will know that this is here for consistency, and for the ability to add a method that would winnow down the allowed params further (as is done in the other controllers).

I admit that at first the `alias` confused me until I saw the context of the rest of the PR.